6_Deploy_roles_login.md

```diff
@@ -12,12 +12,14 @@ Add support for:
 * User logins, only logged in users should be able to use the application;
 * user roles:
     * a read-only role - that can only see data;
-    * an admin role - that can see and add data;
+    * an admin role - that can see and add data.
+* a user management console where admin users can create users and manage user roles;
```

Comment:
Do we want to use the word console? Could be confusing with `console.log`?
Dashboard? Area? Section? Page?